nix/project.nix

```diff
@@ -48,24 +48,21 @@ let
             # the test executable, which are actually set up right (we have a
             # build-tool-depends on the executable we need)
             # I'm slightly surprised this works, hooray for laziness!
-            plutus-metatheory.components.tests.test1.preCheck =
+            plutus-metatheory.components.tests.test-simple.preCheck =
               let
-                cmp = config.hsPkgs.plutus-metatheory.components.tests.test1;
+                cmp = config.hsPkgs.plutus-metatheory.components.tests.test-simple;
                 deps = cmp.executableToolDepends;
               in
               ''PATH=${lib.makeBinPath deps}:$PATH'';
 
-            # FIXME: Somehow this is broken even with setting the path up as above
```

Comment:
@zeme-wana I think this is what I have to remove to re-enable these tests in CI, but that's a guess.  I'm also not sure what the stuff above about `preCheck` is for or whether we still need it.

Reply:
Yes, the test2->test-detailed came back and is passing.
As for the `preCheck`, it *might* be obsolete. 
I commented it out, let's see what CI thinks